### PR TITLE
merge phobic and pthash

### PR DESCRIPTION
**Pull request by stefanfred**

### CODE REVIEW

**Comment by stefanfred** [comment]
note: currently only examples compiles ans works for alpha=1

File: include/builders/util.hpp [hunk 1]
```diff
@@ -309,4 +333,4 @@ struct hash_generator {
     uint64_t m_seed;
 };
 
-}  // namespace pthash
+}  // namespace pthash
```
Comment (ByteHamster) [inline include/builders/util.hpp:336]: Unnecessary change removing the last line break

**Review by ByteHamster — COMMENTED** [review]
The changes are less extreme than I feared :+1:

**Comment by jermp** [comment]
Just merged into `dev`. Let's see :)

**Comment by jermp** [comment]
> note: currently only examples compile and work for alpha=1

Now all examples work, even with alpha < 1. Tomorrow, I'll look into make the `build` executable compile too. The issue is that currently `dense_partitioned_phf` does not support external mem building.

Another issue is: we have to resolve the ugly `needsFreeArray` and `Minimal` bool templates :-D
